data-processing/scripts/run_pipeline.py

```diff
@@ -377,9 +377,19 @@ def run_commands_for_arxiv_ids(
     if args.store_log:
         command_classes.append(StorePipelineLog)
 
+    # Determine what commands should be run.
     filtered_commands = []
 
+    # Determine the entities that will be processed. If no entities were specified,
+    # then set the entities to the default (or empty if a list of commands was provided).
+    entities: List[str] = []
+    if args.entities is not None:
```

Comment:
Just nitting... `[]` is False in Python when used in an `if` statement (behaves like None). See my more extensive note below. Typically it's more Pythonic to say simply `if args.entities:` which will be True when `len(args.entities) > 0`